lib/router.js

```diff
@@ -510,7 +510,12 @@ class Router {
         // added to the higher level spec?
         Object.assign(scope.specRoot.definitions, spec.definitions);
         Object.assign(scope.specRoot.securityDefinitions, spec.securityDefinitions);
-        scope.specRoot.tags = scope.specRoot.tags.concat(spec.tags || []);
+        scope.specRoot.tags = scope.specRoot.tags.concat(spec.tags || [])
+            .filter((tag, index, self) => {
+                return index === self.findIndex((t) => {
+                    return t.name === tag.name  && t.description === tag.description;
```

Comment:
Having the same tag names with different descriptions in the same file is kinda an error. Let's throw an exception if that occurs.

Reply:
Done.